calibration/proposed-tests/test_calibration_findings.py

```
@@ -0,0 +1,238 @@
+        # Pins current behavior; will need to flip when R-4 (min count = 3) lands.
+        with repo_fixture() as repo:
+            block = (
+                "def helper():\n"
+                "    blank_choice = []\n"
+                "    if blank_choice is None:\n"
+                "        blank_choice = [(\"\", \"---\")]\n"
+                "    return blank_choice\n"
+            )
+            (repo / "src" / "a.py").write_text(block, encoding="utf-8")
+            (repo / "src" / "b.py").write_text(block.replace("def helper", "def helper2"), encoding="utf-8")
+            code, data = check_json(repo)
```

Ruff (0.15.12)

[warning] Unpacked variable `code` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)